### ISSUE TITLE
Monkey King's tree dance does not always show available trees consistently

### ISSUE THREAD

**Issue by roseannepark1102** [jeffhill/Dota2#293]
### Description

Sometimes it shows all the trees, sometimes it shows certain ones and sometimes it shows none at all that monkey can jump to. dont know the cause and cant replicate the bug in demo but happens what feels like every monkey game.

**Comment by WasBooking** [comment]
Yes, sometimes a tree (not broken) is highlighted, but you cannot jump on it, because there is not enough range

**Comment by real-vaLor** [comment]
Similar issue as #445 with tree indicator being bugged. I notice this in all of my matches but also cannot replicate it in demo. It can be seen in this highlight of qojqva's monkey king game at 2:42 (12:02 ingame time): https://youtu.be/Eb_MVi90zCY?t=162 .
